.github/workflows/build-icon.yml

```diff
@@ -0,0 +1,48 @@
+name: build icon via webhook
+
+on:
+  workflow_dispatch:
```

Comment:
ref: https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#workflow_dispatch